addons/mail/static/src/emoji_picker/emoji_picker.js

```diff
@@ -131,6 +131,7 @@ export class EmojiPicker extends Component {
     shouldScrollElem = null;
 
     setup() {
+        window.aku = this;
```

Comment:
Missing the comment that explains that it's for a `window.aku.state.searchStr = 'burger'` script.

Reply:
Oops, de-🍔fied